### DISCUSSION TITLE
[Feature] Hide Album from partner sharing

### DISCUSSION THREAD

**Discussion by maxheadroom** [immich-app/immich#6720] [Feature Request]
### The feature

As a user I would like to hide a specific album from partner sharing. While family/partner sharing of all pictures is great, I'd like to keep certain pictures private to my account. 

As a user I would know this feature has been implemented when:
* I can create an Album and put pictures/videos (assets) into this album
  * the assets which are placed in this album will not be shared to other accounts I've shared with as partner accounts
* there is an option in the settings where I can specify to exempt one (or more) of my existing albums from partner sharing
* if the assets are also member of other albums which are not exempted from partner sharing, the assets should still not be shared in those not restricted albums. Not-Sharing should have precedence 
* optionally a small indicator icon on an asset would be nice to indicate its sharing has been exempted.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2451
